spec/features/comment_posts_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+require 'create_user_and_sign_up'
+
+RSpec.feature "Comment", type: :feature do
+  scenario "User can comment on a post" do
+    create_user_and_sign_up
+    click_link 'New post'
+    fill_in 'post_caption', with: 'Cute cat'
+    page.attach_file('post_image', Rails.root + "spec/fixtures/stanley.jpg")
+    click_button 'Create post'
+    fill_in 'comment_body', with: 'Your cat is super cute'
+    click_button 'Add'
```

Comment:
consider rewriting in a more declarative style, e.g creating a method like:
```
def add_comment(text)
  fill_in 'comment_body', with: 'Your cat is super cute'
  click_button 'Add'
end
```

and similar for creating the post in the first place. see review guidelines: https://github.com/Hives/instagram-challenge/blob/master/docs/review.md#prefer-declarative-feature-tests